nvidia/lib

```diff
@@ -7,18 +8,6 @@ U_USERLAND="${U_OS%%/*}"
 
 ARCH_TRIPLET="${U_MACHINE}-${U_KERNEL,,}-${U_USERLAND,,}"
 
-NVIDIA_SUPPORT_DISABLED="$(snapctl get nvidia-support.disabled)"
-
-NVIDIA_SUPPORT_CLASSIC="false"
-[ -L "/var/lib/snapd/hostfs/usr/lib/${ARCH_TRIPLET}/libcuda.so" ] && NVIDIA_SUPPORT_CLASSIC="true"
-
-if snapctl is-connected gpu-2404 || snapctl is-connected graphics-core22; then
-    NVIDIA_SUPPORT_CORE="true"
-else
-    NVIDIA_SUPPORT_CORE="false"
-fi
-
-
```

Comment:
NVIDIA_SUPPORT_CLASSIC and NVIDIA_SUPPORT_CORE are used in snap/hooks/post-refresh after sourcing this script.